se3cnn/SO3.py

```diff
@@ -5,6 +5,7 @@
 Using ZYZ Euler angles parametrisation
 """
 import math
+from appdirs import user_cache_dir
```

Comment:
`appdirs` is not installed by default with anaconda. shall we add some `try: except:` that set `user_cache_dir` to the identity function in case `appdirs` is not installed?

Reply:
add to requirements, maybe? it is 12 kB standalone, and saves a lot of headache

Reply:
ok

Reply:
added